app/javascript/components/admin/instances/card.js

```diff
@@ -0,0 +1,105 @@
+import Deploy from './actions/deploy'
+import Restart from './actions/restart'
+import Stop from './actions/stop'
+import Delete from './actions/delete'
+import axios from 'axios'
+
+export default {
+  name: 'InstanceCard',
+
+  props: {
+    instance: Object
+  },
+
+  methods: {
+    updateStatus (data) {
+      this.status = data.status
+      this.processing = data.processing
+    }
+  },
+
+  data () {
+    return {
+      status: this.instance.status,
+      processing: false
+    }
+  },
+
+  render () {
+    return (
+      <div class='col-lg-4 col-sm-6 mb-4'>
+        <div class={`card card-instance-details border-${this.status.level}`}>
+          <div class='card-header'>
+            <div class='d-flex'>
+              <h5 class='card-title flex-grow-1 m-0'>
+                <a href={this.instance.url} class={`text-${this.status.level}`}>
+                  {this.instance.name}
+                </a>
+              </h5>
+              <div class='dropdown'>
+                <a
+                  class='dropdown-toggle'
+                  type='button'
+                  id={`openode-instance-${this.instance.id}`}
+                  data-toggle='dropdown'
+                  aria-haspopup='true'
+                  aria-expanded='false'
+                >
+                  <i class='fa fa-cog' />
+                </a>
+                <div
+                  class='dropdown-menu dropdown-menu-right'
```

Comment:
The "deploying" thing is nice, however i will not have this feature for now. A deployment can take max 5 minutes (worst case scenario), and so it's too long for an HTTP query. Later there could be websocket (action cable). The status can be a link to the list of deployments

![image](https://user-images.githubusercontent.com/8007722/73585795-b8772d00-4472-11ea-87aa-89a84bdce729.png)

Reply:
What about this: The request returns the status immediately, so it will be quick. I can do a live poll every 15 or 30 seconds to a GET /admin/instances/:id/deployments/:id/status and update accordingly. Then in the future replace that with action cable.

Reply:
Yes that would be good if it's not too complicated. only one thing, make sure to verify maximally during x minutes (example 5 minutes), it must not go out of control and for example keep doing it if you are no more in the page.

Reply:
queued status is great